l10n - added Czech translation into xml

p-bo opened the pull request:
Added short and long description in Czech language, as well as keywords.

p-bo committed:
l10n - added Czech translation into xml

Added short and long description in Czech language, as well as keywords.

amma35 commented on additionalalerts.xml:
```diff
@@ -6,10 +6,12 @@
     <logo>https://raw.githubusercontent.com/InfotelGLPI/additionalalerts/master/additionalalerts.png</logo>
     <description>
         <short>
+            <cs><![CDATA[Posílání dalších upozornění. Tento zásuvný modul umožňuje posílat upozornění e-mailem ohledně nových strojů v OCS-NG, nezinventarizované počítače za X dnů, položky s nevyplněným datem nákupu a náplních do tiskáren, které docházejí.]]></cs>
```
Can you add <cs> after <en>, please?

amma35 commented on additionalalerts.xml:
```diff
@@ -6,10 +6,12 @@
             <en><![CDATA[Sending of additional alerts. This plugin enables you to send email alerts regarding the new machines inventoried from OCS-NG, the not inventoried computers from X days, the items with empty buy date and Cartridges whose level is low.]]></en>
         </short>
         <long>
+            <cs><![CDATA[Tento zásuvný modul umožňuje posílat e-mailem doplňková upozornění:<br />- Nové stroje zinventarizované v OCS-NG (mající stav určený do nastavovacího modulu OCS-NG),<br />- Počítače nezinventarizované za X dnů (Volba podporovaného stavu: aby nebyly brány v úvahu stroje na skladě),<br />- Položky s nevyplněným datem zakoupení (Volba podporovaných typů: aby nebylo v úvahu b ...
```
Can you add <cs> after <en>, please?

p-bo replied:
Sorry, suddenly started to place it only alphabetically. Then realized, that it is driving order of panels with description in pages on https://plugins.glpi-project.org site. So, switched to schema where tag with description is en first (because rest of displayed page is), then by alphabetical order, to easier orientation for reader/editor. With tags lang and keywords then alphabetically only, because they seems are not influencing anything.
Is this correct please?

amma35 replied:
The new languages must be added afterwards to respect the order of all the plugins so <fr>, <en> and the new languages

p-bo replied:
Must admit, that I cannot understand the rest of sentence after word afterwards.
What is meant with "order of all plugins" in this case context?

Please, what is wrong with idea, that on plugins.glpi-project.org, there tabs with languages of description should be in alphabetical order, except for English (to display by default because rest of page is in English).

And order in list of supported languages should be alphabetical too (see for fr_FRen_GBde_DEes_ESit_ITpl_PLpt_BRru_RUtr_TRes_ARet_EEfi_FIhe_ILpt_PTsl_SIzh_TW in https://plugins.glpi-project.org/#/plugin/fusioninventory – it is very difficult for reader to locate her/his language without resorting to browser search function).

See, I don't want make problems, I will place description, language codes and tags in whatever order you want in xml data, but I'm unhappy about resulting bad experience for plugins.glpi-project.org visitors, which is resulting from not doing alphabetical ordering in presenting information to them.

I want to continue to work on translating of descriptions for another plugins for plugins.glpi-project.org, so please lets try to resolve this issue (maybe open discussion across team to have some general resolution?).

Thanks :)
Pavel

amma35 replied:
The modifications we asked you to make are not related to tags.
You made no modification to change this order: 
![translate](https://user-images.githubusercontent.com/15138371/44339211-d9b3ab80-a480-11e8-9af5-4d53453dabd7.png)

The change we ask you is the order of the tabs : 
![onglet](https://user-images.githubusercontent.com/15138371/44339259-fcde5b00-a480-11e8-84b0-3fcf9df8797c.png)

p-bo replied:
Regarding language codes, I attempted to correct order in other plugins, where I was requested to add this as a last line too and don't fix order.

So what should be the order of tabs? I assume EN, to be default (because rest of page is in English); then alphabetically, so for example CS, DE, ES, FR – am I right please?

amma35 replied:
You just have to add after those already existing as you did here: https://github.com/InfotelGLPI/webapplications/commit/af702b9063d252aa78f8fc44b42fc8c75b38defb

p-bo replied:
Well, I do see – but result is, that visitor is reading English page text, then suddenly there is some text in french, then (s)he cannot locate desired language, because order is not alphabetical, the same on some plugins pages for supported languages and result is bad user experience only because unfortunate presentation of existing data. This is point I tried to improve along the adding localization too…

p-bo committed:
Tweaked additionalalerts.xml

Ordered alphabetically for correct sorting when displayed on plugins.glpi-project.org
Revised list of languages to present only translations, that are at least from 50% done on Transifex (to don't confuse users about true availability in their languages).
Minor tweaks of Czech translation.

p-bo committed:
Tweaked additionalalerts.xml

Ordered alphabetically for correct sorting when displayed on plugins.glpi-project.org
Revised list of languages to present only translations, that are at least from 50% done on Transifex (to don't confuse users about true availability in their languages).
Minor tweaks of Czech translation.

p-bo reviewed (commented):
Ordered alphabetically for correct sorting when displayed on plugins.glpi-project.org
Revised list of languages to present only translations, that are at least from 50% done on Transifex (to don't confuse users about true availability in their languages).
Minor tweaks of Czech translation.

p-bo commented:
Ping :)

keha35 commented:
Hi,

We will not add other languages description.
It will be difficulte for us to maintain more langugages.

Thanks.
